src/Models/Constants/Constants.ts

```diff
@@ -93,7 +93,7 @@ export const LOCAL_STORAGE_KEYS = {
             showExplorer: 'cardboard.feature.dataHistoryExplorer' // enables data history explorer feature if supported
         },
         Proxy: {
-            forceProxy: 'cardboard.feature.forceProxy' // force proxy to run instead of CORS
+            forceCORS: 'cardboard.feature.forceCORS' // force CORS to run instead of proxy
```

Comment:
is there a reason why we changed the key name in local storage? Also, does this require small migration? But since it is a feature flag I guess it should be good.

Reply:
Yes, I changed the logic to be default proxy and force CORS if needed. I thought it was better to change the key in local storage to reflect that change in logic.